modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path

```diff
@@ -20,6 +20,11 @@
 		<td>//button[contains(@class,'btn btn-sm btn-secondary')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>ADD_TAB_BUTTON</td>
+		<td>//button[contains(@class,'btn-secondary') and contains(.,'Add Tab')]</td>
+		<td></td>
+	</tr>
```

Comment:
Do you use this path in some steps of the test? You can use something like: `LexiconEntry.gotoAddNoSelectFrame();` to add tab on the layout

Reply:
Done !